crates/node/src/constants.rs

```diff
@@ -14,4 +14,10 @@ pub const ERC721_CLASS_HASH: &str = "0x80000";
 pub const UDC_CONTRACT_ADDRESS: &str = "0x041a78e741e5af2fec34b695679bc6891742439f7afb8484ecd7766661ad02bf";
 pub const UDC_CLASS_HASH: &str = "0x90000";
 
+pub const SINGLE_EVENT_EMITTING_CONTRACT_ADDRESS: &str = "0x041a78e741e5af2fec34b695679bc6891742439f7afb8484ecd7766661ad02cf";
+pub const SINGLE_EVENT_EMITTING_CLASS_HASH: &str = "0x91000";
+
+pub const MULTIPLE_EVENT_EMITTING_CONTRACT_ADDRESS: &str = "0x041a78e741e5af2fec34b695679bc6891742439f7afb8484ecd7766661ad02cf";
```

Comment:
the two addresses are the same, does it really work?

Reply:
this is the node constants, not the mock. i erroneously added. will remove